apps/jobbrowser/src/jobbrowser/api2.py

```diff
@@ -169,27 +169,42 @@ def profile(request):
     response['status'] = 0
     return JsonResponse(response)
 
-
-@api_error_handler
-def query_store_api(request, path=None):
+def query_store_proxy(request, path=None):
```

Comment:
nit: is this is an util and not a real view, adding a prefix to show it is good practice, e.g. `_query_store_proxy`

Reply:
Thanks, done.